### ISSUE TITLE
Postprocess part

### ISSUE THREAD

**Issue by ByronHsu** [rookiepig/BinaryStereo#3]
Do you implement the postprocess part?

**Comment by rookiepig** [comment]
yes some simple filtering based methods.

**Comment by ByronHsu** [comment]
Sorry but the postprocess part is empty in BinaryStereo.cpp.
Do you implement it in other files?

**Comment by rookiepig** [comment]
Yeah. In this project [https://github.com/rookiepig/CrossScaleStereo](https://github.com/rookiepig/CrossScaleStereo)
